### DISCUSSION TITLE
Εξεταστική Σεπτεμβρίου

### DISCUSSION THREAD

**Discussion by P2019SARAKATSANIS** [courses-ionio/sw#1769] [Q&A] [answered]
Καλησπέρα σας,

Υπάρχει κάποιος που γνωρίζει την διαδικασία της εξεταστικής του Σεπτεμβρίου σε αυτό το μάθημα ;

(Έχω τα περισσότερα παραδοτέα έτοιμα και περίμενα για κάποια ανακοίνωση για να τελειώσω ότι έκανα και να δω αν θα υπάρχουν αλλαγές, ώστε να γίνει pull request)

@epidrome

**Comment by iwnasFilippou** [comment]
Μέχρι πότε είναι η διορία για την εξεταστική σεπτεβρίου @epidrome ;

**Comment by epidrome** [comment, marked as answer]
https://epidrome.github.io/teaching/faq/
